packages/mendel-middleware/index.js

```diff
@@ -24,6 +24,23 @@ function MendelMiddleware(opts) {
     return function(req, res, next) {
         req.mendel = req.mendel || {};
 
+        req.mendel.getBundleEntries = function() {
+            return Object.keys(trees.bundles).reduce(
```

Comment:
I think we can cache the result of this. No need to do this for every single request.